AUDIO/PlayPain.md

```diff
@@ -5,27 +5,52 @@ ns: AUDIO
 
 ```c
 // 0xBC9AE166038A5CEC 0x874BD6CB
-void PLAY_PAIN(Ped ped, int painID, int p1);
+void PLAY_PAIN(Ped ped, int painID, float p3, bool createNetEvent);
 ```
 
-```
-Needs another parameter [int p2]. The signature is PED::PLAY_PAIN(Ped ped, int painID, int p1, int p2);  
-Last 2 parameters always seem to be 0.  
-EX: Function.Call(Hash.PLAY_PAIN, TestPed, 6, 0, 0);  
-Known Pain IDs  
-________________________  
-1 - Doesn't seem to do anything. Does NOT crash the game like previously said. (Latest patch)  
-6 - Scream (Short)  
-7 - Scared Scream (Kinda Long)  
-8 - On Fire  
-```
+## Description
+Plays a pain sound. A maximum of 33 pain IDs are allowed.  
+Below is a list of all the pain IDs (Asterisks indicate that sounds can play in a Low, Medium or High fashion), for example: `PAIN_LOW_GENERIC`. The corresponding pain strings belong to the game exe.
 
-```
-NativeDB Added Parameter 4: Any p3
-```
+## Pain IDs:
```

Comment:
Please do not use custom `##` headers anywhere, they probably won't end up working correctly for codegen.

Reply:
Yup, gone. Let me know if this is ok later, so I can squash the PR, thanks!